src/ontology/mondo-edit.obo

```diff
@@ -459200,18 +459200,16 @@ property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/596/albright
 
 [Term]
 id: MONDO:0022414
-name: allain-babin-demarquez syndrome
-comment: Reason of obsoletion: out of scope - MONDO:excludeObsoleteSource. Term to consider: -
-subset: obsoletion_candidate
+name: obsolete allain-babin-demarquez syndrome
 subset: orphanet_rare {source="Orphanet:1526"}
-subset: rare
 synonym: "acro cephalo synostosis" RELATED [GARD:0000124]
 synonym: "craniosynostosis synostoses hypertensive nephropathy" RELATED [GARD:0000124]
-xref: Orphanet:1526 {source="MONDO:equivalentObsolete", source="GARD:0000124"}
-is_a: MONDO:0002254 {source="https://orcid.org/0000-0002-6601-2165"} ! syndromic disease
+xref: Orphanet:1526 {source="GARD:0000124", source="MONDO:obsoleteEquivalentObsolete"}
+property_value: IAO:0000231 OMO:0001000 {source="MONDO:excludeObsoleteSource"}
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7190" xsd:anyURI
-property_value: IAO:0006012 "2024-04-01" xsd:string
 property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/124/allain-babin-demarquez-syndrome" xsd:anyURI {source="GARD:0000124"}
```

Comment:
@sabrinatoro should the `seeAlso` property be retained since this GARD entity is obsolete?

Reply:
I'll remove it